mission_controller:
  ros__parameters:
    # Navigation timeouts
    nav_timeout_sec: 120.0
    
    # Actuator timeouts
    servo_timeout_sec: 10.0
    pump_timeout_sec: 30.0
    
    # Recovery settings
    enable_recovery: true
    max_recovery_attempts: 3
